### DISCUSSION TITLE
Following the installation process of `https://shadcn-docs-nuxt.vercel.app/getting-started/installation`, what is the problem?

### DISCUSSION THREAD

**Discussion by llds66** [ZTL-UwU/shadcn-docs-nuxt#52] [Q&A]
![image](https://github.com/user-attachments/assets/c2038eaa-1895-416f-af7d-662d1972021e)

**Comment by puzzle-it-nu** [comment]
I have had the same problem and I think it might have todo with mixing pnpm and npm or by using --shamefully-hoist with pnpm which fixed it for me.
Have look at your node_modules folder. My guess is that there are only a handful of folders instead of tens.
